### DISCUSSION TITLE
No verified college email

### DISCUSSION THREAD

**Discussion by Aniket1026** [community/community#49899] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello everyone, 
I'm trying to apply for github student developer pack but In the process it wants me to add a email given by the college. But in my case we're not given any email ID from the college . So how can i apply for the student pack? 
Thank you 😊

**Comment by bkpecho** [comment]
Hello brother @Aniket1026 

If your college doesn't provide you with an email address, you can still apply for the GitHub Student Developer Pack by using your **personal email address or requesting an email alias from your school's IT department**. 

You can also try to provide alternative documentation to prove your enrollment in a qualifying institution. This could include **a tuition bill, a transcript, or a letter from the registrar's office**.

If neither of those options work, check out this [blog post](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/#:~:text=You%20may%20be%20prompted%20for,that%20demonstrates%20your%20current%20enrollment) or [video](https://www.youtube.com/watch?v=8grritqx5io).

I hope my answer is helpful and I wish you the best of luck with your application!

**Comment by Aniket1026** [comment]
Hello brother @bkpecho 
Thanks for replying and sharing the useful resources. 
I applied using my personal email and in the proof I uploaded my college ID which had the dates of my overall session from 2020 to 2024 . But my request has been declined twice by saying that : - Attendees of this school must verify and use their academic email address.
So what should I do in such case
